conans/test/functional/toolchains/cmake/test_cmake_toolchain.py

```diff
@@ -555,6 +555,20 @@ def test_cmake_toolchain_runtime_types_cmake_older_than_3_15():
     assert "LIBCMTD" in client.out
 
 
+@pytest.mark.skipif(platform.system() != "Windows", reason="Only for windows")
+# @pytest.mark.tool("cmake", "3.28")
+def test_cmake_toolchain_winsdk_version():
+    client = TestClient(path_with_spaces=False)
+    client.run("new cmake_lib -d name=hello -d version=0.1")
+    # cmake = client.load("CMakeLists.txt")
+    # cmake = cmake.replace("cmake_minimum_required(VERSION 3.15)",
+    #                       "cmake_minimum_required(VERSION 3.27)")
+    # client.save({"CMakeLists.txt": cmake})
+    client.run("create . -s arch=x86_64 -s compiler.version=193 "
+               "-c tools.microsoft:winsdk_version=8.1")
+    assert "Conan toolchain: CMAKE_GENERATOR_PLATFORM=x64" in client.out
```

Comment:
In the CMake lists we can print the value of `CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION` - which is a read-only variable of the version of the SDK that CMake selected - although I'm not 100% what happens if you choose a version that you don't have installed

Reply:
If you pick a version that you don't have installed, with the OLD policy, it will fallback to the latest SDK

Reply:
I'll add that check, good idea.